tests/containers/podman_ipv6.pm

```diff
@@ -75,6 +76,12 @@ sub run {
     # Test container IPv6 connectivity
     #   there is iptables masquarade so the container appears under the host address
     assert_script_run("podman exec -it test-ipv6 curl -sSf6 -o /dev/null $curl_test https://opensuse.org");
+
+    # Check if a IPv6-only container can be reached
+    $image = 'registry.opensuse.org/opensuse/nginx:latest';
+    script_retry("podman pull $image", timeout => 300, delay => 15, retry => 3);
+    assert_script_run("podman run --name http-ipv6 --network podman-ipv6 --ip6 fd00:c0de:ba5e::3 -d -p 8080:80 $image", timeout => 180);
+    script_retry("curl -v http://127.0.0.1:8080/", delay => 15, retry => 3);
```

Comment:
Maybe it's overkill, but we can add `-4` to be extra sure:

```suggestion
    script_retry("curl -4 -v http://127.0.0.1:8080/", delay => 15, retry => 3);
```

Reply:
Podman by default creates IPv6 NAT. I'll create both `curl -4` and `curl -6` and will let you know.

Reply:
So:
```bash
# curl -4v http://127.0.0.1:8080/; echo StbBp-$?-
*   Trying 127.0.0.1:8080...
* Connected to 127.0.0.1 (127.0.0.1) port 8080
<h1>Welcome to nginx!</h1>
* Connection #0 to host 127.0.0.1 left intact
StbBp-0-
# curl -6v http://127.0.0.1:8080/; echo VMV2j-$?-
* Closing connection
curl: (7) Couldn't connect to server
VMV2j-7-
```
what do you think @ricardobranco777? Is this expected?

Reply:
> So:
> 
> ```shell
> # curl -4v http://127.0.0.1:8080/; echo StbBp-$?-
> *   Trying 127.0.0.1:8080...
> * Connected to 127.0.0.1 (127.0.0.1) port 8080
> <h1>Welcome to nginx!</h1>
> * Connection #0 to host 127.0.0.1 left intact
> StbBp-0-
> # curl -6v http://127.0.0.1:8080/; echo VMV2j-$?-
> * Closing connection
> curl: (7) Couldn't connect to server
> VMV2j-7-
> ```
> 
> what do you think @ricardobranco777? Is this expected?

Yes.  You must use an IPv6 address with `-6`.

Reply:
```
# curl -6v http://[::1]:8080/; echo Q8DBv-$?-
*   Trying [::1]:8080...
* connect to ::1 port 8080 from ::1 port 49530 failed: Connection timed out
* Failed to connect to ::1 port 8080 after 133479 ms: Couldn't connect to server
* Closing connection
curl: (28) Failed to connect to ::1 port 8080 after 133479 ms: Couldn't connect to server
Q8DBv-28-
```
still ...

Reply:
Please recheck